tomb

```diff
@@ -185,6 +189,27 @@ _is_found() {
 	return $?
 }
 
+# Track acces and modification time of tomb files.
+# $1: file to track
+# date format: seconds since Epoch
+# stat format: <last access>:<last modified>
+_track_stat() {
+    local file="$1"
+    local stat=$(stat --format="%X:%Y" "$file")
+    TOMBFILESSTAT+=("$file" "$stat")
+}
+
+# Restore files stats
+_restore_stat() {
+    local file stat
+    for file stat in "${(@kv)TOMBFILESSTAT}"; do
```

Comment:
nice! I did not know destructuring key/values in iterations over maps worked this way also in zsh.
I have restarted the build on the CI BTW, investigating why the error.